.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Run integration tests
+
+on:
+  pull_request:
+    types: [opened,reopened,synchronize]
+
+jobs:
+  test-unit-jest:
```

Comment:
Minor, should we rename this to `test-intergration-jest`?

Reply:
🤦‍♂️